src/main/java/com/automationexercise/Pages/ProductsPage.java

```diff
@@ -0,0 +1,54 @@
+package com.automationexercise.Pages;
+
+import com.automationexercise.DriverManager.DriverManager;
+import com.automationexercise.Utilities.Utility;
+import com.automationexercise.Utilities.WaitsUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.interactions.Actions;
+
+import static com.automationexercise.DriverManager.DriverManager.getDriver;
+
+public class ProductsPage {
+    /*  Selectors Sections  */
+    private By cartNavbarElement = By.cssSelector("[class='nav navbar-nav'] li:nth-child(3) a");
+    private By blueTopItem = By.xpath("//*[@class='features_items']/div[2]");
+    private By menTshirtItem = By.xpath("//*[@class='features_items']/div[3]");
+    private By bluetopaddtoCart = By.cssSelector("[data-product-id='1']");
+    private By menTshirtAddtocart = By.cssSelector("[data-product-id='2']");
+    private By continueShoppingButton = By.cssSelector("[data-dismiss='modal']");
+
+    /*  Behaviour Sections  */
+    public CartPage cartNavbarElementClick(){
```

Comment:
name the clicking functions to be ClickOnX.
X > the button that you will click on.